cmd/node/main.go

```diff
@@ -238,7 +238,7 @@ VERSION:
 		Value: logger.LogInfo,
 	}
 	// bootstrapRoundIndex defines a flag that specifies the round index from which node should bootstrap from storage
-	bootstrapRoundIndex = cli.UintFlag{
+	bootstrapRoundIndex = cli.Uint64Flag{
```

Comment:
you can also change in main.go line 696 -> change `GlobalUint` to `GlobalUint64` and remove the cast

Reply:
ok